FrontEnd/src/components/ProfilePage/FormComponents/FormFields/PasswordField.js

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import { PropTypes } from 'prop-types';
+import EyeInvisible from '../../../authorization/EyeInvisible';
+import EyeVisible from '../../../authorization/EyeVisible';
+import css from './PasswordField.module.css';
+
+const PasswordField = (props) => {
+    const [showPassword, setShowPassword] = useState(false);
+
+    const togglePassword = () => {
+        setShowPassword(!showPassword);
+    };
+
+    return (
+        <div className={css['password-field__item']}>
+            <div className={css['password-field__label-wrapper']}>
+                {props.requredField &&
+                <span>
+                    *
+                </span>}
+                <label
+                    htmlFor={props.inputId}
+                >
+                    {props.label}
+                </label>
+            </div>
+            <div className={css['password-field__password']}>
+                <div className={css['password-field__password__wrapper']}>
+                    <input
+                        id={props.inputId}
+                        name={props.name}
+                        type={showPassword ? 'text' : 'password'}
+                        value={props.value}
+                        onChange={props.updateHandler}
+                        placeholder={props.label}
+                        required={props.requredField}
+                    />
+                </div>
+                <span
```

Comment:
Using a `<span>` for the password visibility toggle is not ideal because spans are not inherently focusable or interactive elements. Consider using a `<button>` instead, which is more appropriate for actions and improves accessibility
e.g.
```
<button className={css['password-visibility']} onClick={togglePassword} type="button">
   {!showPassword ? <EyeInvisible /> : <EyeVisible />}
</button>
```

Reply:
I agree with this point. I added a span there because it's done that way in our LoginPage component.